example/routes/index.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:dart_frog/dart_frog.dart';
+import 'package:google_cloudevents_dart/events/cloud/firestore/v1.dart';
+import 'package:http/http.dart' as http;
```

Comment:
I wonder if we should provide an example without HTTP as a dependency.

IIRC, the logic looks something like this
```dart
  final bytes = await context.request.bytes().fold(<int>[], (result, chunk) => [...result, ...chunk]);
  final event = DocumentEventData.fromBuffer(Uint8List.fromList(bytes));
```